src/frontend/classic/components/MangaViewer.svelte

```diff
@@ -0,0 +1,217 @@
+<script lang="ts">
+    import type {
+        IMediaContainer,
+        IMediaChild,
+    } from "../../../engine/providers/MediaPlugin";
+    import { Loading } from "carbon-components-svelte";
+    import WideViewerImage from "./WideViewerImage.svelte";
+    import { onMount } from "svelte";
+    import { preloadImage } from "../utils/image";
+    import { inversedReading, doublePage } from "../utils/storage";
+
+    type ChapterImages = { src: string; nextSrc: string | undefined };
+
+    export let item: IMediaContainer;
+    export let throttlingDelay: number;
+    export let imagePadding: number;
+    export let currentImageIndex: number;
+
+    let chapterImages: ChapterImages[] | undefined = undefined;
+    const carbonLgBreakpoint = 1056;
+    const carbonMaxBreakpoint = 1584;
+    const minImageWidth = 50;
+    const maxImageWidth = 100;
+    let imageWidth: number;
+
+    const handleKeyDown = (evt: KeyboardEvent) => {
```

Comment:
Use Named function instead of Anonymous functions (more readable).
Unlesss there is a point to do so ?

Same comment for all functions

Reply:
Yeah, there is no reason, I'm just used to it because I worked on a lot of react project ^^